packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -594,6 +601,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       'styles': classNames,
       'checked': this.props.checked,
       'disabled': this.props.disabled,
+      'allowDisabledFocus': this.props.allowDisabledFocus,
```

Comment:
consider getting allowDisabledFocus, checked, and disabled when this.props is initially accessed near the top of this function